README.md

```diff
@@ -1,46 +1,49 @@
 # ANDELA SOCIALS
 
-[![CircleCI](https://circleci.com/gh/AndelaOSP/Andela-Socials.svg?style=shield&circle-token=f63d8e7f6a1e65078cd1988847cbb29e6382bba1)](https://circleci.com/gh/AndelaOSP/Andela-Socials)
+[![CircleCI](https://circleci.com/gh/AndelaOSP/Andela-Socials.svg?style=shield&circle-token=f63d8e7f6a1e65078cd1988847cbb29e6382bba1)](https://circleci.com/gh/AndelaOSP/Andela-Socials) [![Maintainability](https://api.codeclimate.com/v1/badges/0da1c9a1fec63a855e21/maintainability)](https://codeclimate.com/github/AndelaOSP/Andela-Socials/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/0da1c9a1fec63a855e21/test_coverage)](https://codeclimate.com/github/AndelaOSP/Andela-Socials/test_coverage)
 
-[![Maintainability](https://api.codeclimate.com/v1/badges/0da1c9a1fec63a855e21/maintainability)](https://codeclimate.com/github/AndelaOSP/Andela-Socials/maintainability) [![Test Coverage](https://api.codeclimate.com/v1/badges/0da1c9a1fec63a855e21/test_coverage)](https://codeclimate.com/github/AndelaOSP/Andela-Socials/test_coverage)
+Andela Social was born from the desire to keep Andelans sociable within the organization.
+It is a platform for events aggregation, to get Andelans from different departments, cohorts and centers to mingle and have fun together over a cup of coffee, a game of soccer, at a swimming pool or at a friends dinner party.
+With this platform, events organized within the fellowship are properly tracked and managed for all to see and for all to relate with irrespective of their centers.
+This will improve socialization among Andelans both inside and outside the organisation.
 
-Andela Social was born out of the desire to keep Andelans very sociable within the organization.
-It is a platform for events aggregation, to get Andelans from different departments to mingle and have fun together over a cup of coffee, a game of soccer, at a swimming pool or at a friends dinner party.
-It is a platform to get events organized within the fellowship properly tracked and managed for all to see and for all to relate with irrespective of their centers.
-These will improve socialization between Andelans inside and outside the company.
+## PROJECT REQUIREMENTS
+- Node
+- Postgres Database
```

Comment:
Please add the correct spelling and version number e.g. `PostgreSQL v. 10.9`

Reply:
This is ambiguous. Node has different meanings. Please specify the version and correct terminology i.e. `NodJS 8.x.x`

Reply:
There are other project requirements such as Python, virtualenv etc.